src/Profile/Rule.cs

```diff
@@ -16,20 +16,22 @@ public Rule(ConfigNode node)
 			interval = Lib.ConfigValue(node, "interval", 0.0);
 			rate = Lib.ConfigValue(node, "rate", 0.0);
 			ratio = Lib.ConfigValue(node, "ratio", 0.0);
-			degeneration = Lib.ConfigValue(node, "degeneration", 0.0);
-			variance = Lib.ConfigValue(node, "variance", 0.0);
+            input_threshold = Lib.ConfigValue(node, "input_threshold", 0.0);
```

Comment:
What happened with indenting here? Check rest of file too.

Reply:
looks fine in Visual studio and notepad++

Reply:
original code was tab indented, new code is space ind

Reply:
ahh makes sense, @steamp0rt changed all the spacing to tabs not so long ago and my visual studio is set as spaces >.<